_toc.yml

```diff
@@ -1,42 +1,62 @@
 - file: index
 - chapters:
-  - file: community/index
-    sections:
-    - file: community/mission_and_values
-    - file: community/team
-    - file: community/code_of_conduct
-    - file: community/code_of_conduct_reporting
-    - file: community/governance
-    title: Community
+  - file: tutorials/start_index
+    sections: 
+    - file: tutorials/fundamentals/installation
+    - file: tutorials/absolute_beginners
+    - file: tutorials/fundamentals/getting_started
+    - file: plugins/stable/find-and-install-plugin 
+    - file: tutorials/fundamentals/viewer
+    title: Getting started
   - file: tutorials/index
     sections:
-    - file: tutorials/fundamentals/index
-      sections:
-      - file: tutorials/fundamentals/installation
-      - file: tutorials/fundamentals/getting_started
-      - file: tutorials/fundamentals/viewer
-      - file: tutorials/fundamentals/image
-      - file: tutorials/fundamentals/labels
```

Comment:
I think we somehow lost a lot of the fundamentals tutorial content from the TOC? Eg: fundamentals/image, labels, points, shapes, surface, tracks, vectors.

Edit: wait, no. It's all here, just moved into a 'layers' subsection. I'd guess that this hides the content a little more than we'd like - it seems easier to accidentally skip over now. New users might be likely to click on a too advanced topic because "layers" doesn't sound like a thing they need to know about (whereas "segmentation" does sound familiar)

Reply:
I think that's a valid concern @GenevieveBuckley, but I do see that logically they group nicely under that section. I know we're planning on having a "Layers at a glance" document introducing and comparing the different layer types. Maybe having that at top level in "Getting started" would help drive home the importance?

Reply:
I like both those ideas

Reply:
That sounds reasonable, thanks!

Reply:
I'm not really sure how to implement this right now. Initially i thought one good idea would be to include the Layers subdocuments twice, once in the Getting started section, and another in the How-tos section. However this causes the Layers document to be highlighted twice in the table of contents when the page is visited:

![Screenshot_20211004_150510](https://user-images.githubusercontent.com/3949932/135901693-7b4001ce-b4df-401b-b318-bfc58f1ff956.png)

I'll see if I can think of another solution.

Reply:
Just approved it!